services/frontend-service/src/ui/components/ReleaseCard/ReleaseCard.scss

```
@@ -58,6 +58,7 @@
 }
 
 .release-environment {
+  white-space: nowrap;
   background-color: grey;
   .mdc-evolution-chip__text-label {
```

```suggestion
  .mdc-evolution-chip__text-name {
```
this className doesn't seem to be used anywhere. If changed to this, it will turn the label in the chips back to white as it was originally (I'm not sure if them turning grey was intended)

---

fixed it (with next push)
I kept the classes but just for testing.
I moved the css rules to the parent object, because actually I want the style of both the name and the numbers to be the same.